index.html

```diff
@@ -741,6 +738,40 @@
         </section>
 
         <section>
+          <h2 id='ldpcvget'>HTTP GET</h2>
+          <p>
+            An <a>LDPCv</a> MUST respond to <code>GET Accept: application/link-format</code> as
+            indicated in [[!RFC7089]]
+            <a href='https://tools.ietf.org/html/rfc7089#section-5'>section 5</a> and specified in
+            [[!RFC6690]] <a href='https://tools.ietf.org/html/rfc6690#section-7.3'>section 7.3</a>.
+          </p>
+          <p>
+            An implementation SHOULD respond with the <code>Allow</code> header as outlined in 
+            <a href='#ldpcvoptions'></a>.
+          </p>
+          <p>
+            An implementation MUST respond with a <code>Content-Type</code> header.
+          </p>
+          <p>
+            If an <a>LDPCv</a> supports <code>POST</code>, then it MUST respond with 
```

Comment:
I would be ok removing this redundancy with: 4.3.5 HTTP POST

Reply:
I can remove this, I had included it since it seemed that an implementer would look to the section about GET or HEAD requests to determine which headers would be included in those responses.  When reviewing the specification I found I would miss components that needed to be included in certain interactions because the details of them were listed in other API request sections.